### PR TITLE
fix playing video in slideshow, fullscreen video not allowed in modal…

### PR DESCRIPTION
**Pull request by FernetMenta**
fix http://trac.kodi.tv/ticket/16792

@phil65 @BigNoid @ronie 
In addition window MyPic needs a video control or it can't show videos in slide show. Currently the window only displays thumbs.

### CODE REVIEW

**Comment by FernetMenta** [comment]
jenkins build this please

File: addons/skin.estuary/1080i/Includes.xml [hunk 1]
```diff
@@ -990,11 +990,11 @@
 			<control type="videowindow">
 				<depth>DepthBackground</depth>
 				<include>FullScreenDimensions</include>
-				<visible>Player.HasVideo</visible>
+				<visible>Player.HasVideo + !Slideshow.IsActive</visible>
```
Comment (FernetMenta) [inline addons/skin.estuary/1080i/Includes.xml]: @phil65 is this correct?

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by ronie** [comment]
could you add this one as well
https://github.com/ronie/xbmc/commit/9ac749344c74a49acdc4eeaf6a3038f37865c028

**Comment by FernetMenta** [comment]
@ronie will do
btw: there is a similar issue in confluence MyPVRChannels.xml. video is rendered twice when this window is active because background is included.

now I know why the minimized view always required so many resources and made video stutter.

**Comment by ronie** [comment]
> btw: there is a similar issue in confluence MyPVRChannels.xml. video is rendered twice when this window is active because background is included.

^ @HitcherUK 

i'll post a heads-up in the skin dev forum, pretty sure every skin has similar issues.

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by HitcherUK** [comment]
> btw: there is a similar issue in confluence MyPVRChannels.xml. video is rendered twice when this window is active because background is included.
> ^ @HitcherUK
> 
> i'll post a heads-up in the skin dev forum, pretty sure every skin has similar issues.

Are you sure this is the case because checking the code for background video has this condition in it -
`<visible>Player.HasVideo + !Skin.HasSetting(ShowBackgroundVideo) + !Window.IsVisible(TVChannels) + !Window.IsVisible(RadioChannels)</visible>`

**Comment by FernetMenta** [comment]
@HitcherUK seems I missed this. Then it should be ok, though not really nice having the background dependent on particular windows.

**Pull request merged**